src/chains/ethereum/ethereum/src/api.ts

```diff
@@ -2884,6 +2886,198 @@ export default class EthereumApi implements Api {
       overrides
     );
   }
+
+  /**
+   * Returns a collection of historical block gas data and optional effective fee spent per unit of gas for a given percentile of block gas usage.
+   * Added in [EIP-1559](https://www.alchemy.com//blog/eip-1559)
```

Comment:
We have an `@EIP` JSDoc tag for this. Let's link to the EIP repo instead of alchemy. We've been putting these links before the `@example` section.

It be something like:

```
   * @EIP [1559 - Fee market change](https://github.com/ethereum/EIPs/blob/master/EIPS/eip-1559.md)
   * 
   * @example
```

Reply:
:+1: